src/api/app/models/group.rb

```diff
@@ -25,11 +25,6 @@ class Group < ApplicationRecord
   validates :title,
             uniqueness: { message: 'is the name of an already existing group.' }
 
-  # groups have a n:m relation to user
-  has_and_belongs_to_many :users, -> { distinct }
```

Comment:
``But then you need a has_many :users, through: :groups_users``
Otherwise how do you get the users?
(same for roles...)

Reply:
Right, forgot to add it.

Reply:
Cleaned it up now. And added the roles relation again. There we actually need it.